src/cogent3/core/sequence.py

```diff
@@ -1090,6 +1092,111 @@ def annotate_matches_to(self, pattern, annot_type, name, allow_multiple=False):
 
         return annot
 
+    def _repr_html_(self):
+        html = self.to_html(limit=self._repr_policy["num_pos"],)
+        return html
+
+    def to_html(
```

Comment:
This method looks like a near duplicate of the corresponding method on `AlignmentI`. This tells me we should move the alignment method code into a function inside `cogent3.format.alignment`. Call the function `rich_html`. 

Then both alignment and sequence `to_html()` methods call that function.

Reply:
Similarly, would we need a cogent3.format.sequence?

Reply:
Are you referring to the rich_html function in table.py?